app/Http/Controllers/User/HomeController.php

```diff
@@ -6,15 +6,18 @@
 use App\Http\Controllers\Controller;
 use Auth;
 use App\Http\Requests\LoginRequest;
-use App\Repositories\Contracts\UserInterface;
+use App\Repositories\Contracts\UserRepository;
+use App\Services\Services;
 
 class HomeController extends Controller
 {
     protected $user;
+    protected $services;
 
-    public function __construct(UserInterface $postCustomer)
+    public function __construct(UserRepository $user, Services $services)
     {
         $this->user = $user;
```

Comment:
```suggestion
        $this->userRepository = $userRepository;
```
viết tnay cho đỡ nhầm lẫn